src/utils/updateNickname.ts

```diff
@@ -0,0 +1,27 @@
+import { DISCORD_BASE_URL } from "../constants/urls";
+import { env } from "../typeDefinitions/default.types";
+
+export async function updateNickName(
+  discordId: string,
+  nickname: string,
+  env: env
+) {
+  const changeNickNameURL = `${DISCORD_BASE_URL}/guilds/${env.DISCORD_GUILD_ID}/members/${discordId}`;
+  const data = { nick: nickname };
+  try {
+    const nameChangeResponse = await fetch(changeNickNameURL, {
+      method: "PATCH",
+      headers: {
+        "Content-Type": "application/json",
+        Authorization: `Bot ${env.DISCORD_TOKEN}`,
+      },
+      body: JSON.stringify(data),
+    });
+    if (nameChangeResponse.ok) {
+      return await nameChangeResponse.json();
+    }
+    return "Some internal Server error occurred";
```

Comment:
Can we write this in constant instead of hard coding the error message?

Reply:
Done and updated with the PR